<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Experience - MO DEHESTANI</title>
    <link rel="stylesheet" href="styles.css">
</head>
<body>
    <header>
        <h1>Professional Experience</h1>
        <nav>
            <a href="index.html">Home</a>
            <a href="skills.html">Skills</a>
            <a href="education.html">Education</a>
            <a href="publications.html">Publications</a>
            <a href="contact.html">Contact</a>
        </nav>
    </header>
    <main>
        <section>
            <h2>Computational Biologist & Genomic Data Scientist</h2>
            <p><strong>German Center for Neurodegenerative Diseases DZNE, Tübingen</strong></p>
            <p><em>11.2024 – Present</em></p>
            <ul>
                <li>Provide bioinformatics, biostatistics, and omics data analytic support within the core facility.</li>
            </ul>
        </section>

        <section>
            <h2>Postdoctoral Research Fellow</h2>
            <p><strong>Biomedical Data Science & Machine Learning Group (Bansal Lab), German Center for Neurodegenerative Diseases DZNE, Tübingen</strong></p>
            <p><em>07.2023 – 10.2024</em></p>
            <ul>
                <li>Developed and optimized <strong>scMultiflow</strong>: a Dockerized pipeline for reproducible multi-omics single-cell analysis (scRNA-seq, scATAC-seq), enhancing data integration, scalability, and processing efficiency.</li>
                <li>Co-supervised graduate students in statistical genomics.</li>
                <li>Contributed to the <strong>Michael J. Fox Foundation’s FOUNDIN-PD initiative</strong> as part of the bioinformatics team.</li>
                <li>Contributed to Development of an automated Python-based tool for multiomics QC analysis (<strong>scMoMsQC</strong>).</li>
                <li>Identified early drivers in LRRK2 PD pathogenesis within the oligodendrocytic lineage by integrating single-cell multiomics data with GWAS data, collaborating with wet lab scientists to validate findings.</li>
                <li>Taught the course <strong>"Analytic Methods in Genomics, Epigenomics, and Transcriptomics"</strong> to Master's students in Cell/Molecular Neuroscience at the GTC Graduate School, University of Tübingen.</li>
            </ul>
        </section>

        <section>
            <h2>Doctoral Research Fellow</h2>
            <p><strong>Department of Neurodegenerative Diseases (Gasser Lab), Hertie Institute for Clinical Brain Research, University Hospital Tübingen</strong></p>
            <p><em>10.2018 – 06.2023</em></p>
            <ul>
                <li>Customized scalable single-cell and bulk data analysis pipelines.</li>
                <li>Developed a novel post-GWAS workflow integrating single-cell omics with polygenic scoring for predictive modeling of Parkinson’s outcomes in a local cohort using advanced statistical methods.</li>
                <li>Applied genome-wide analyses and machine learning to large cohorts (UKBB, COURAGE-PD, AMP-PD) to investigate genetic risk and resilience factors in Parkinson’s disease.</li>
                <li>Conducted statistical/population genomics analyses, including GWAS, genotyping QC, eQTL mapping, polygenic scoring, cohort stratification, Mendelian randomization, heritability analysis, and burden testing.</li>
            </ul>
        </section>

        <section>
            <h2>Research Intern</h2>
            <p><strong>Institute of Clinical Genetics (Di Donato Lab), Carl Gustav Carus Hospital of TU Dresden</strong></p>
            <p><em>06.2018 – 09.2018</em></p>
            <ul>
                <li>Trained in clinical bioinformatics, NGS data interpretation and variant prioritization for mental retardation syndromes.</li>
            </ul>
        </section>

        <section>
            <h2>Research Assistant</h2>
            <p><strong>Medical Genetics Research Center MGRC, Mashhad University of Medical Sciences</strong></p>
            <p><em>01.2014 – 04.2017</em></p>
            <ul>
                <li>Focused on neurogenetics, researching molecular brain development and the genetics of inherited epilepsies.</li>
            </ul>
        </section>
    </main>
    <footer>
        <p>&copy; 2024 MO DEHESTANI</p>
    </footer>
</body>
</html>
